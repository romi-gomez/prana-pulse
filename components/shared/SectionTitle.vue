<template>
    <div class="my-4">
      <h2 class="text-2xl font-semibold text-mainDark">{{ title }}</h2>
    </div>
</template>
  
<script setup>
  import { defineProps } from 'vue'
  
  const props = defineProps({
    title: {
      type: String,
      required: true
    }
  })
</script>
  
<style scoped>
 </style>