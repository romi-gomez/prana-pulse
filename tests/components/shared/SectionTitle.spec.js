import { mount } from '@vue/test-utils'
import SectionTitle from '@/components/shared/SectionTitle.vue'

describe('SectionTitle', () => {
  test('renders correctly', () => {
    const wrapper = mount(SectionTitle, {
      props: {
        title: 'Test Title'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })

  test('displays the correct title', () => {
    const wrapper = mount(SectionTitle, {
      props: {
        title: 'Test Title'
      }
    })
    expect(wrapper.text()).toBe('Test Title')
  })
})