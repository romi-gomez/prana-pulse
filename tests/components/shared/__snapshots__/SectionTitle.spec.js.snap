// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle renders correctly 1`] = `
<div
  class="my-4"
>
  <h2
    class="text-2xl font-semibold text-mainDark"
  >
    Test Title
  </h2>
</div>
`;
